repo2docker/buildpacks/julia/julia_project.py

```diff
@@ -13,32 +16,20 @@ class JuliaProjectTomlBuildPack(PythonBuildPack):
     # ALL EXISTING JULIA VERSIONS
     # Note that these must remain ordered, in order for the find_semver_match()
     # function to behave correctly.
-    all_julias = [
-        "0.7.0",
-        "1.0.0",
-        "1.0.1",
-        "1.0.2",
-        "1.0.3",
-        "1.0.4",
-        "1.0.5",
-        "1.1.0",
-        "1.1.1",
-        "1.2.0",
-        "1.3.0",
-        "1.3.1",
-        "1.4.0",
-        "1.4.1",
-        "1.4.2",
-        "1.5.0",
-        "1.5.1",
-        "1.5.2",
-        "1.5.3",
-    ]
+    @property
```

Comment:
What about combining this with `@functools.lru_cache(maxsize=1)`? It would skip HTTP requests on repeated look up. We can't use `functools.cache` because it is too new :(